txtdirect.go

```diff
@@ -301,6 +301,12 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 		return nil
 	}
 
+	if host == "127.0.0.1" {
```

Comment:
That only partially solves the general problem.
What about any other used IP etc.?
We should handle IPs correctly either way so that it always either goes right back to the instance redirect or 404. What do you think?